_cucumber/cucumber.rake

```diff
@@ -27,7 +27,13 @@ task :features do |t, args|
     profile = 'parallel'
   end
 
-  exit_status = runner.run(profile)
+  if ENV['CUCUMBER_TAGS'].to_s.empty?
+    tags = nil
+  else
+    tags = ENV['CUCUMBER_TAGS']
```

Comment:
You raise a good point there, as if there was a typo in the cucumber tag, for example @smke 0 tests would be executed with an exit code of 0, meaning they would pass. I've added a check in there was verifies the tag is within a known good tags array. New tags can be added or removed from there.